docs/blog/2024-06-14-project-pythia-mystmd.md

```diff
@@ -0,0 +1,117 @@
+The Project Pythia infrastructure team has faced several challenges in maintaining their infrastructure that has hindered scalability of the project as well as maintenance burden on the team:
+
+Configuration Sharing
+: Updating shared configuration elements like navigation and branding requires changes to each individual cookbook, creating a time-consuming process where the infrastructure team opens many similar pull-requests to each individual cookbook to update the site as a whole.
+
+Maintenance Burden
+: Customizing the theme for Pythia was completed using the Sphinx framework and inheritance of multiple themes, many of which have been changed over the past few years, and this has placed a high maintenance burden on the infrastructure team to keep everything up to date.
+
+Content Reuse Limitations
+: There is currently limited ability to reuse and reference content from the foundational resources across different cookbooks, and referencing a concept requires cookbook readers to jump out of one tutorial and into another (e.g. how to use Conda or Jupyter). There are also many shared components, such as license information, “how to cite this cookbook”, and managing readme updates.
+
+Limited use of MyST in Cookbooks
+: Currently there is limited use of any advanced features of MyST Markdown in the cookbook tutorials (e.g. admonitions, cross-references, etc.) as the main way to experience a cookbook is through JupyterLab, and the default renderer of Jupyter doesn’t interpret MyST Markdown (but [jupyterlab-myst](https://github.com/executablebooks/jupyterlab-myst) does!).
+
+Throughout the week, the core ExecutableBooks team, [Curvenote](https://curvenote.com), and [2i2c](https://2i2c.org) helped address some of these constraints with the new mystmd tooling that we have been developing over the last several years.
+
+## Architecture and Workflow
+
+One of the most exciting aspects of the transition was that it was almost painless, we have gone to great lengths to ensure that content is supported and backwards compatible, and there were almost no content changes required. The transition of a cookbook was almost as easy as:
+
+```bash
+conda install mystmd
+myst
+```
+
+Over the coming months we will roll out targeted upgrade routines for JupyterBook that understand and upgrade the `_config.yml`, but where we were this week was that required manual effort by the team. It was pleasantly surprising to see that this also fixed some issues out of the box!
+
+:::{figure} ./images/pythia-jb-myst.png
```

Comment:
Here's a later version of this screenshot I posted after we fixed the Project Pythia logo:

<img width="1723" alt="Screenshot 2024-06-12 at 14 30 18" src="https://github.com/executablebooks/meta/assets/45105935/dea73bcf-d1db-4007-979e-c6ca19df5007">

Reply:
Thank you! Updated!